rpm/xrootd-multiuser.spec

```
@@ -12,22 +12,23 @@ URL: https://github.com/bbockelm/xrootd-multiuser
 Source0: %{name}-%{version}.tar.gz
 
 %define xrootd_current_major 4
+%define xrootd_current_minor 12
 %define xrootd_next_major 5
 
 BuildRoot: %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
-BuildRequires: xrootd-server-libs >= 1:%{xrootd_current_major}.0.0-0
-BuildRequires: xrootd-server-libs <  1:%{xrootd_next_major}.0.0-0
-BuildRequires: xrootd-server-devel >= 1:%{xrootd_current_major}.0.0-0
-BuildRequires: xrootd-server-devel <  1:%{xrootd_next_major}.0.0-0
+BuildRequires: xrootd-server-libs >= 1:%{xrootd_current_major}
+BuildRequires: xrootd-server-libs <  1:%{xrootd_next_major}
+BuildRequires: xrootd-server-devel >= 1:%{xrootd_current_major}
+BuildRequires: xrootd-server-devel <  1:%{xrootd_next_major}
```

```suggestion
BuildRequires: xrootd-server-libs = 1:%{xrootd_current_major}
BuildRequires: xrootd-server-devel = 1:%{xrootd_current_major}
```

Can't we simplify the 4 requirements down to 2?

---

Nop, it looks like if you use "=" you have to specify the full version major.minor.x and unless we want to change the spec every time we have a new version of xrootd, I don't think we can do this change.